vcd/resource_vcd_org_vdc_test.go

```diff
@@ -36,6 +36,19 @@ func TestAccVcdOrgVdcReservationPool(t *testing.T) {
 		// cause vDC ignores empty values and use default
 		"MemoryGuaranteed": "1",
 		"CpuGuaranteed":    "1",
+		// These parameters for Flex allocation.
+		// Part of HCL created dynamically and with empty values we don't create Flex part
```

Comment:
Suggestion for here and other occurrences in the file:
```suggestion
		// Part of HCL is created dynamically and with empty values we don't create the Flex part:
```

Reply:
changed